content/docs/user-guide/gems/reference/ros2/concepts-and-components-overview.md

```diff
@@ -0,0 +1,111 @@
+---
+linkTitle: Guide
+title: O3DE ROS 2 Concepts and Structure
+description: Understanding basic concepts and structure of O3DE ROS 2 Gem
```

Comment:
Don't need to add O3DE in the title to save on text space (though I know some topics do this). It's most important to mention O3DE, expanding it to "Open 3D Engine (O3DE), in the description. The description is used for SEO purposes.
```suggestion
linkTitle: Concepts and Structure
title: ROS 2 Concepts and Structure
description: Understanding basic concepts and structure of the ROS 2 Gem in Open 3D Engine (O3DE).
```